google-workspace.sgnl.yaml

```diff
@@ -0,0 +1,277 @@
+description: "Google Workspace as a System of Record"
+address: "admin.googleapis.com" 
+defaultSyncFrequency: HOURLY
+defaultSyncMinInterval: 1
+defaultApiCallFrequency: SECONDLY
+defaultApiCallMinInterval: 1
+type: "GoogleWorkspace-1.0.0"
+# Example Config: 
+# {
+#   "apiVersion": "v1",
+#   "domain": "sgnldemos.com",
+#   "filters": {
+#     "user": {
+#       "showDeleted": true,
+#       "query": "isAdmin=false"
+#     },
+#     "group": {
+#       "query": "adminCreated=true"
+#     },
+#     "member": {
+#       "includeDerivedMembership": false,
+#       "roles": "MEMBER"
+#     }
+#   }
+# }
+
+# Adapter Config Field Info:
+# [Required] APIVersion (String): The version of the Google Workspace API to use. Currently, only v1 is supported.
+
+# DISCLAIMER: Exactly one of either 'Customer' or 'Domain' must be set in the adapter config.
+# Read more about the Customer and Domain parameters here: https://developers.google.com/admin-sdk/directory/v1/guides
+
+# [Required/Optional] Customer (String): The unique ID for the customer's Google Workspace account. In case of a multi-domain account,
+# to fetch all entities for a customer, use this field instead of domain.
+
+# [Required/Optional] Domain (String): Use this field to get entities from only one domain.
+
+# [Optional] Filters (Map): These filters can be applied to the endpoints that are used to retrieve each entity. 
+
+# For more information on the User endpoint: https://developers.google.com/admin-sdk/directory/reference/rest/v1/users/list
+# [Optional] Filters.UserFilters (Map): These filters can be applied to the User endpoint.
```

Comment:
```suggestion
# [Optional] filters.user (Object): These filters can be applied to the User endpoint.
```

I'd refrain from using golang terminology here. Use JSON terminology as the config is in JSON

Reply:
fixed